Resources/Prototypes/_StarLight/Entities/Structures/Walls/walls.yml

```
@@ -85,4 +85,38 @@
     base: state
   - type: Appearance
   - type: Ricochetable
-    chance: 1
\ No newline at end of file
+    chance: 1
+
+- type: entity
+  parent: BaseWall
+  id: WallConcrete
+  name: conrete wall
+  suffix: Reinforced
+  components:
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: StructuralMetallicStrong
+  - type: Sprite
+    sprite: _Starlight/Structures/Walls/concrete_wall.rsi
+  - type: Icon
+    sprite: _Starlight/Structures/Walls/concrete_wall.rsi
+    state: full
+  - type: IconSmooth
+    key: walls
+    base: state
+  - type: Appearance
+  - type: Ricochetable
```

```suggestion
  - type: Appearance
  - type: Pierceable
    level: Rock
  - type: Ricochetable
```
But getting a shot through it is way harder.